content/applications/websites/ecommerce/checkout_payment_shipping/shipping.rst

```diff
@@ -0,0 +1,71 @@
+========
+Shipping
+========
+
+Depending on your shipping strategy, you have the choice to either use your :ref:`own shipping
+methods <ecommerce-own-shipping>`, or use an integration with an :ref:`existing shipping provider
+<ecommerce-shipping-providers>`.
+
+Own shipping methods
+====================
+
+.. _ecommerce-own-shipping:
+
+You can create your own custom shipping methods and define rules to compute shipping costs. To do
+so, go to :menuselection:`Website --> Configuration --> Shipping Methods`, and either select an
+**existing** shipping method, or :guilabel:`Create` one. When creating a shipping method, you can
+choose between
+:doc:`Fixed Price </applications/inventory_and_mrp/inventory/shipping/setup/delivery_method>`,
+:doc:`Based on Rules </applications/inventory_and_mrp/inventory/shipping/setup/delivery_method>`,
+and :guilabel:`Pickup in store`.
+
+Pickup in store
+---------------
+
+:guilabel:`Pickup in store` must first be **enabled** in the settings (:menuselection:`Website -->
+Configuration --> Settings --> Shipping section)` by checking :guilabel:`On Site Payments &
+Picking`. Once enabled, you can select and :guilabel:`Customize Pickup Sites`. The
+:guilabel:`Picking sites` are **website-specific**, although the same :guilabel:`picking site` can
+be enabled for **multiple** websites.
```

Comment:
Actually, they _can_ be website-specific but they don't have to be.

I guess that explaining that shipping methods can be (multi-)website-specific is the important piece of info here, especially if you move the "pickup in store" part to its inventory page where it belongs. But then you already have a dedicated section for this.